src/botPage/view/shared.js

```diff
@@ -36,6 +36,17 @@ export const appendRow = (trade, state) => ({
     ],
 });
 
+export const appendRowTop = (trade, state) => ({
```

Comment:
Since we are repeating codes in this methid and the upper one, is there a better way to mix them?

Reply:
We can maybe mix them with conditional rendering using ternary operators.

Reply:
Good job, I think we can only decide on ` ...state.rows,` ... 
Whatever you think is better